### PR TITLE
Add wg-stories to build-system/ OWNERS.

### PR DESCRIPTION
**Pull request by gmajoulet**
Add @ampproject/wg-stories to `build-system/` OWNERS.

### CODE REVIEW

**Comment by gmajoulet** [comment]
Context: https://github.com/ampproject/amphtml/pull/34955#issuecomment-866146299

**Review by rsimha — APPROVED** [review]

**Comment by gmajoulet** [comment]
@ampproject/wg-infra can you plz help me understand why the CircleCI OWNERS test fails? I got this multiple times in a row :(

```
[21:58:02] Starting 'check-owners'...
[21:58:05] SUCCESS: Parsed build-system/compile/OWNERS successfully; produced 5 rule(s).
[21:58:05] ERROR: build-system/test-configs/forbidden-terms.js is not an OWNERS file.
[21:58:05] Finished 'check-owners' after 2s
```

**Comment by estherkim** [comment]
#34986 was fixed a few hours ago. plz rebase :(

**Review by estherkim — APPROVED** [review]

File: build-system/compile/OWNERS [hunk 1]
```diff
@@ -20,6 +20,7 @@
         {name: 'ampproject/wg-components'},
         {name: 'ampproject/wg-infra'},
         {name: 'ampproject/wg-performance'},
+        {name: 'ampproject/wg-stories'},
```
Comment (rcebulko) [inline build-system/compile/OWNERS:23]: Retroactive, but at this point would it make sense to just set it as `{name: '*'}`?

**Pull request merged**